README.md

```diff
@@ -596,6 +596,7 @@ Options:
   * Should probably run the build in a docker container anyway
 * `yarn install`
 * `yarn build`
```

Comment:
Should this just say `MAPBOX_ACCESS_TOKEN=your-token-here yarn build`

Reply:
probably 👍